stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -134,6 +152,14 @@ public static Token retrieveToken(
         for (String propertyName : propertyNames) {
             propertyMap.put(propertyName, System.getProperty(propertyName));
         }
+
+        propertyMap.put("phone.brand", Build.BRAND);
```

Comment:
I'm assuming it's okay to tag our data with new stuff that might be useful for us?